lib/modules/platform/github/index.spec.ts

```diff
@@ -542,6 +543,23 @@ describe('modules/platform/github/index', () => {
       ).rejects.toThrow(REPOSITORY_NOT_FOUND);
     });
 
+    it('throws unexpected graphql errors', async () => {
+      httpMock
+        .scope(githubApiHost)
+        .post(`/graphql`)
+        .reply(200, {
+          errors: [
+            {
+              type: 'RATE_LIMITED',
+              message: 'API rate limit exceeded for installation ID XXXXXXX.',
+            },
+          ],
+        });
+      await expect(
+        github.initRepo({ repository: 'some/repo' })
+      ).rejects.toThrow(PLATFORM_UNKOWN_ERROR);
```

Comment:
I would prefer you return the rate limited error when you _know_ that's what it is, and the unknown error only when there's an unknown response